src/scripts/main.js

```diff
@@ -1 +1,37 @@
-'use strict';
+const track = document.querySelector('.carousel__track');
+const slides = Array.from(track.children);
+const nextButton = document.querySelector('.carousel__button-right');
+const prevButton = document.querySelector('.carousel__button-left');
+
+const slideWidth = slides[0].getBoundingClientRect().width;
+
+const setSlidePosition = (slide, index) => {
+  slide.style.left = slideWidth * index + 'px';
+};
+
+slides.forEach(setSlidePosition);
+
+window.addEventListener('resize', (e) => {
+  slides.forEach(setSlidePosition);
+});
+
+// eslint-disable-next-line no-shadow
+const moveToSlide = (track, currentSlide, targetSlide) => {
+  track.style.transform = 'translateX(-' + targetSlide.style.left + ')';
```

Comment:
You could change this notation to:
```
track.style.transform = `translateX(-${targetSlide.style.left})`
```
In this way, instead of concatenating strings, you would be adding a variable to a string.